Mod Menu Helper/assets/modmenu/lang/el_gr.json

```diff
@@ -0,0 +1,53 @@
+{
+  "modmenu.dropInfo.line1": "Οι τροποποιήσεις του FO έχουν  §9✎§r ή §8✎§r",
+  "modmenu.dropInfo.line2": "οι εναλλακτικές έχουν §c*§r, οι υπόλοιπες δεν υποστηρίζονται",
+  "modmenu.summaryTranslation.slight-gui-modifications": "Animations, έκδοση πακέτου στο μενού\n§9✎§8 Animations, χρώματα, εμφάνηση FPS, ...",
+  "modmenu.summaryTranslation.advancementinfo": "Μεγαλύτερη, εξυπνότερη οθόνη επιτευγμάτων\n§9✎§8 Χρώματα, μέγεθος του πεδίου πληροφοριών, ...",
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Μεγαλύτερη οθόνη επιτευγμάτων\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.animatica": "Τα κινούμενα είδη του Optifine\n§8✎ (μόνο εξιδικευμένες επιλογές)",
+  "modmenu.summaryTranslation.antighost": "Διορθώνει τους αόρατους κύβους με το /ghost\n§9✎§8 Πλήκτρο",
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Γρηγορότερα κρεβάτια\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.bettermounthud": "Εμφάνηση φαγητού-εμπιρείας πάνω σε άλογο\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.bobby": "Ανοιχτά φορτώμενα κομμάτια server\n§9✎§8 Κύβοι-οντότητες, παραμόνη cache μνήμης, ...",
```

Comment:
Cache? server?

Reply:
cache goes for προσωρινή μνήμη, but it captures too much space, and I had to find a way to make it fit in

Reply:
for server too